amethyst_renderer/src/prelude.rs

```diff
@@ -1,19 +0,0 @@
-//! Contains common types that can be glob-imported (`*`) for convenience.
```

Comment:
Convention seem to be the prelude containing traits,  i.e. things that are used implicitly, but not directly instantiated.

Reply:
I heard that, yeah, but I think that many libraries have different preludes with different usages.